src/Articles/tests/components/AllArticles.test.jsx

```diff
@@ -0,0 +1,41 @@
+/* eslint-disable no-undef */
+import React from "react";
+import { shallow, mount } from "enzyme";
+import {
+  AllArticles,
+  mapDispatchToProps,
+  mapStateToProps
+} from "../../components/AllArticles";
+import getAllArticles from "../../redux/actions/AllArticlesActions";
+
+const props = {
+  getArticles: jest.fn(),
+  items: []
+};
+
+describe("test the component", () => {
+  it("matchs the component snapshot", () => {
+    const mycomponent = shallow(<AllArticles {...props} />);
+    expect(mycomponent).toMatchSnapshot();
+  });
+
+  it("should match the snapshot", () => {
+    const wrapper = mount(<AllArticles {...props} />);
+    wrapper.setProps(props);
+    expect(wrapper).toBeDefined();
+  });
+});
+
+describe("test the map state to props and dispatch", () => {
+  it("matches the mapstostate", () => {
+    expect(
+      mapStateToProps({
+        allArticleReducer: { is_retrieving: false, items: [] }
+      })
+    ).toEqual({ is_retrieving: false, items: [] });
+  });
+  it("match map state to dispatch", () => {
+    const dispatch = jest.fn();
+    expect(mapDispatchToProps(dispatch)).toHaveProperty("getArticles");
+  });
+});
```

Comment:
Also, test that indeed we call the getArticles function when component mounts and updates

Reply:
@AGMETEOR I have tested that and added a few more tests to increase coverage thank you.